api/admin/controller/individual_admin_settings.py

```diff
@@ -38,8 +38,6 @@ def process_post(self):
         # For readability: the person who is submitting the form is referred to as "user"
         # rather than as something that could be confused with "admin" (the admin
         # which the user is submitting the form in order to create/edit.)
-
```

Comment:
The comment above this should maybe be moved now that user is no longer in this method?